sippy-ng/src/build_clusters/BuildClusterDetails.js

```diff
@@ -13,13 +13,16 @@ export default function BuildClusterDetails(props) {
       <Grid container spacing={3} alignItems="stretch">
         <Grid item md={12} sm={12}>
           <Card elevation={5} style={{ padding: 20, height: '100%' }}>
-            <Typography variant="h6">Job Runs On {props.cluster}</Typography>
+            <Typography variant="h6">
+              Periodic Job Runs On {props.cluster}
+            </Typography>
             <JobRunsTable
               pageSize={10}
               hideControls={true}
               filterModel={{
                 items: [
                   filterFor('cluster', 'equals', props.cluster),
+                  filterFor('name', 'starts with', 'periodic'),
```

Comment:
There are some periodics that start with `release-` or `promote-` unfortunately.  I think you could do ` not(filterFor('name', 'starts with', 'pull'))` to be more accurate

Reply:
Will a regex match be possible for `*periodic*`?

Reply:
The periodics that start with release or promote don't contain periodic anywhere in the name, but I believe all presubmits begin with `pull-`

Reply:
@deepsm007 we probably could, but if I understand correctly, we'd have to plumb a new regex-style `Operator` and logic to implement it [here](https://github.com/openshift/sippy/blob/cebacf96c35f9c540cea8348c8f94cb662383b8b/pkg/filter/filterable.go#L64).